.github/workflows/e2e-embed.yml

```diff
@@ -33,8 +33,8 @@ jobs:
       STRIPE_PRIVATE_KEY: ${{ secrets.CI_STRIPE_PRIVATE_KEY }}
       STRIPE_CLIENT_ID: ${{ secrets.CI_STRIPE_CLIENT_ID }}
       STRIPE_WEBHOOK_SECRET: ${{ secrets.CI_STRIPE_WEBHOOK_SECRET }}
-      PAYMENT_FEE_PERCENTAGE: 0.005
-      PAYMENT_FEE_FIXED: 10
+      STRIPE_PAYMENT_FEE_PERCENTAGE: ${{ secrets.CI_STRIPE_PAYMENT_FEE_PERCENTAGE }}
```

Comment:
We would probably need to set these secrets. Blocking till it's confirmed.

Reply:
@zomars I think we need those

Reply:
I think he can keep them hardcoded in here. They're not affecting the actual app

Reply:
yes